charts/linkerd2-multicluster-remote-setup/templates/service-mirror-rbac.yaml

```diff
@@ -0,0 +1,39 @@
+---
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: {{ .Values.serviceAccountNamespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{.Values.serviceAccountName}}
```

Comment:
I'd like to see some more labels/annotations here so that folks have some documentation around what installed this and what it is used for.